mirrord-layer/src/lib.rs

```diff
@@ -126,14 +138,13 @@ fn init(config: LayerConfig) {
         HOOK_SENDER = Some(sender);
     };
 
-    let enabled_file_ops =
-        ENABLED_FILE_OPS.get_or_init(|| (config.enabled_file_ops || config.enabled_file_ro_ops));
-    let _ = ENABLED_FILE_RO_OPS
-        .get_or_init(|| (config.enabled_file_ro_ops && !config.enabled_file_ops));
-    let _ = ENABLED_TCP_OUTGOING.get_or_init(|| config.enabled_tcp_outgoing);
-    let _ = ENABLED_UDP_OUTGOING.get_or_init(|| config.enabled_udp_outgoing);
+    let enabled_file_ops = ENABLED_FILE_OPS
+        .get_or_init(|| config.feature.fs.is_read() || config.feature.fs.is_write());
+    let _ = ENABLED_FILE_RO_OPS.get_or_init(|| config.feature.fs.is_read());
```

Comment:
These all have default values, right? Do you need get_or_init?

Reply:
None of these have default values in mirrord-layer code, this is the first place we initialize them from values in the config

Reply:
Changed it to `set` function